schema/2.0.0/seurat_encoding.md

```diff
@@ -28,7 +28,7 @@ The following sections describe the individual components of a dataset as encode
 
 ### Data matrix
 
-Matrix data is stored in the slot `assays` under the element `RNA`.
+Matrix data is stored in the slot `assays` under the element `RNA`. Seurat allows elements of `assasy` to have any arbitrary name, but all matrix data from cellxgene will be stored in `RNA`.
```

Comment:
The second sentence seems redundant. 
assasy -> assay 

How about just:

cellxgene stores matrix data in the `RNA` element in the `assays` slot. 

or:

Matfix data from cellxgene is stored in the `RNA` element in the `assays` slot.